guides/common/modules/proc_configuring-a-host-to-use-the-pull-client.adoc

```diff
@@ -2,7 +2,7 @@
 = Configuring a host to use the pull client
 
 For {SmartProxies} configured to use `pull-mqtt` mode, hosts can subscribe to remote jobs using the remote execution pull client.
-Hosts do not require an SSH connection from their {SmartProxyServer}.
+Using the pull client, {SmartProxyServers} do not require an SSH connection to their hosts.
```

Comment:
```suggestion
Using the pull client, {SmartProxies} do not require an SSH connection to their hosts.
```

Reply:
You can also generalize it further: SSH is not involved at all. Perhaps something like
```suggestion
This does not use SSH and hosts do not require opening up any incoming firewall ports.
```

Reply:
Applied Ewoud's suggestion.